modules/ROOT/partials/configuration/skin.adoc

```diff
@@ -1,3 +1,4 @@
+[[skin]]
 == skin
```

Comment:
missing back tick quote, for configuration item name

`skin`

Reply:
done